hackathon_site/review/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from event.models import User
+from registration.models import Application
+
+
+class Review(models.Model):
+    STATUS_CHOICES = [
+        ("Accepted", "Accepted"),
+        ("Waitlisted", "Waitlisted"),
+        ("Rejected", "Rejected"),
+    ]
+
+    reviewer = models.ForeignKey(
+        User, related_name="reviews", on_delete=models.SET_NULL, null=True
+    )
+    application = models.OneToOneField(
+        Application, on_delete=models.SET_NULL, null=True
```

Comment:
This is null=False, since you can't have a review without an application tied to it. I think when I said to make this nullable so we don't have to cascade, I was thinking the foreign key was from application to review, whereas it's actually the other way around.

Tldr; this can be `on_delete=models.CASCADE, null=False`